desktop-exporter/app/components/waterfall-view/duration-bar.tsx

```diff
@@ -12,18 +17,35 @@ type DurationBarProps = {
 };
 
 export function DurationBar(props: DurationBarProps) {
+  const ref = useRef(null);
+  const size = useSize(ref);
+  const labelWidth = 80;
+
   let durationBarColour = useColorModeValue("cyan.800", "cyan.700");
+  let labelTextColour = useColorModeValue("blackAlpha.800", "white");
```

Comment:
I see you brought your "u"s to the party

Reply:
I sure did!